docs/build.gradle

```diff
@@ -85,6 +85,7 @@ testClusters.matching { it.name == "yamlRestTest"}.configureEach {
     keystorePassword 'keystore-password'
     if (BuildParams.isSnapshotBuild() == false) {
```

Comment:
I think this whole block can be removed since we already define the feature flags below. Not sure why this has been duplicated for this flag but it's not needed.